README.md

```diff
@@ -12,6 +14,23 @@ models by incorporating standard components of deep learning recommender models
 example offline
 datasets.
 
+In our initial releases, Merlin Models features a TensorFlow API. The PyTorch API will come later.
+
+## Contents
+
+To learn about the core features of Merlin Models, see the [Models Overview](docs/source/models_overview.md) page.
+
+**[RecSys models zoo](https://nvidia-merlin.github.io/models/main/models_overview.html)** - We provide a high-level API for classic and state-of-the-art deep learning architectures for recommender models. That includes Retrieval (e.g. Matrix Factorization, Two tower, YouTube DNN, ..) and Ranking (e.g. DLRM, DCN-v2, DeepFM, ...) models.
+
+**Building blocks** - Merlin Models is build based on reusable building blocks. That makes it easy to combine those blocks to define new architectures. We have model definition building blocks (MLP layers, Factorization Layers, input blocks, negative samplers, loss functions), training models (data loaders from Parquet files) and evaluation (e.g. ranking metrics).
```

Comment:
Within Merlin Models, recommender models are built based on reusable building blocks, making it easy to combine those blocks to define new architectures.  It provides model definition blocks (MLP layers, Factorization Layers, input blocks, negative samplers, loss functions), training models (data loaders from Parquet files) and evaluation (e.g. ranking metrics).